ruff.toml

```diff
@@ -0,0 +1,28 @@
+select = [
```

Comment:
Mind adding a comment linking to https://beta.ruff.rs/docs/rules/? That should make it faster to find the codes in the future

Reply:
Sure thing.